### ISSUE TITLE
 looks like an insect

### ISSUE THREAD

**Issue by yang4515** [zeit/hyper#3163]
I used the theme apple of oh-my-zsh, the icon of Apple(option+shift+k) became an insect.
<img width="270" alt="screen shot 2018-08-09 at 10 04 52 am" src="https://user-images.githubusercontent.com/21199449/43873988-cba43942-9bbb-11e8-8039-66d3575f10f4.png">

But in xterm or somewhere else, it looks like an apple.
<img width="300" alt="screen shot 2018-08-09 at 10 05 31 am" src="https://user-images.githubusercontent.com/21199449/43873992-cf08b860-9bbb-11e8-937d-0b12dc40d07f.png">

**Comment by albinekb** [comment]
@yang4515 How did you resolve it? Please add your solution before closing so if someone else happens to get into this they can find the fix.

Thanks

**Comment by yang4515** [comment]
@albinekb I customized the font to 'InputMono', which draws the apple into an insect.
<img width="416" alt="screen shot 2018-08-28 at 10 57 32 am" src="https://user-images.githubusercontent.com/21199449/44698138-519d5980-aab1-11e8-996e-777712bd6390.png">
So if the fonts look strange, you can check them out in fontbook.app first.

**Comment by albinekb** [comment]
Then that's correct? I don't see the error if you changed to a font that draws the apple as a bug? @yang4515

**Comment by yang4515** [comment]
Yes, when I changed config.fontFamily in .hyper.js to 'SFMono' or other
fonts, the apple recovered.
This problem is entirely because the font I used is incomplete, I think
other people may not encounter this problem.

Albin Ekblom <notifications@github.com> 于2018年8月28日周二 下午6:50写道：
